src/vuex/modules/session.js

```diff
@@ -239,6 +240,10 @@ export default () =>
       setPreferredCurrency({ state, dispatch }, currency) {
         state.preferredCurrency = currency
         dispatch(`storeLocalPreferences`)
+      },
+      setUserAddressRole({ state, dispatch }, addressRole) {
+        state.addressRole = addressRole
+        dispatch(`storeLocalPreferences`)
```

Comment:
better add this to `persistAddresses`? `storeLocalPreferences` is for users preferences not account specs right

Reply:
Good point! I think better is to remove it as it should not persist in any case, this needs to be checked in every sign in (you can change the controller account at any time, so we need to make sure).